koku/api/settings/tags/mapping/view.py

```diff
@@ -85,9 +85,10 @@ class SettingsTagMappingView(generics.GenericAPIView):
     def get(self, request: Request, **kwargs):
         filtered_qset = self.filter_queryset(self.get_queryset())
         serializer = self.serializer_class(filtered_qset, many=True)
-        paginator = ListPaginator(serializer.data, request)
+        formatted_response = format_tag_mapping_relationship(serializer)
+        paginator = ListPaginator(formatted_response.data, request)
```

Comment:
Since only the data is used from the `Response` object here, it seems like there's little benefit to having `format_tag_mapping_relationship` return a `Response` object. That object is used in the `get` method, but I think it would make `format_tag_mapping_relationship`'s function more clear if it took in data, transformerd it, and returned data rather an `Response`. The caller could decide what to do with the data after it is returned.